torch/distributed/nn/_functions.py

```diff
@@ -0,0 +1,263 @@
+import torch
```

Comment:
Regarding the file name, any reason for using `_functions.py` instead of `functional.py` as done by the `torch.nn` package?

https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py